### ISSUE TITLE
Exact Alpine Sea Holly Table 1 reproduction contract

### ISSUE THREAD

**Issue by shashankskagnihotri** [alirezafarashah/NLCE#4]
Thank you for releasing NLCE. I am preparing an exact, provenance-tracked reproduction of the **Alpine Sea Holly** row in Table 1 from commit `41b22be603abebadad5b0e8ad3fb7e29ef37ff8c`. Could you please clarify or provide the following artifacts/settings?

1. **Exact object command and hyperparameters.** `object_erasure.py` accepts `--beta` and `--gamma`, but the README example passes an unrecognized `--alpha`. Should the Table 1 Alpine run use `--beta 1.0 --gamma 1.0`, the released 850-row `data/config_flowers_alpine.csv`, default pipeline settings, and token floor 20?
2. **Alpine neighbor asset.** The committed `multi_neighbors.json` contains only Camellia, while Appendix Table B lists Alpine Sea Holly neighbors as Metasequoia, Flor de la Mar, Witch-hazel, Sonchus, Honeysuckle, Flora, Astilbe, Catalpa, Mount Sunflower, and Alder. Was that exact list loaded for Table 1? If so, could you provide the original JSON (or confirm that constructing it from the appendix is exact)?
3. **Oxford classifier.** The evaluation notebook uses the placeholder `path/to/flowers_classification_model`. Could you provide the exact fine-tuned ViT-L model/checkpoint, processor, label order, revision/hash, and preprocessing used for the reported 99% classifier and Table 1 `Acc_t`/`Acc_r`?
4. **KID contract.** Could you specify the exact reference image set, feature extractor/implementation, subset size, number of subsets/repeats, and scaling used to report Alpine KID 0.03?
5. **Paper/code implementation differences.** Which behavior produced Table 1?
   - Eq. 2 defines spectrally weighted target projection, but `spectral_w` is computed and not used when constructing `P_F`.
   - The paper writes neighbor-target composition as `P_N P_F`; code uses `P_F P_R`.
   - The paper describes thresholded binary hard erasure; code applies `(1 - mask) * h` using the soft attention map, while `MASK_THR=0.025` is used for attention suppression.
   - The paper describes rewriting key/value projections; code right-projects `encoder_hidden_states` during the second pass.
6. **Generation defaults.** Please confirm scheduler, inference steps, guidance scale, resolution, and whether the exact `diffusers==0.34.0` defaults were used.

We will keep author-release behavior and any paper-equation corrections as separate labeled tracks and will not claim reproduction without the exact evaluator/protocol. Thank you for helping make the comparison faithful.

**Comment by alirezafarashah** [comment]
Thanks for your interest in our project. For Table 1 (Alpine Sea Holly), the run uses the paper's default setup — python object_erasure.py --prompt_file data/config_flowers_alpine.csv --beta 1.0 --gamma 1.0 --demo_mode, the released 850-row data/config_flowers_alpine.csv, default pipeline settings, and δ_token = 20 as stated in Appendix D. The neighbor list is exactly the one in Appendix Table 8 and you can also regenerate it with the released retrieval pipeline (index file + load_targets_and_neighbors), which reproduces the same list. 
For the Oxford classifier, the fine-tuned ViT-L checkpoint isn't hosted on GitHub, please email me at alireza.farashah@mila.quebec and I'll share the model. KID is computed against images generated by the original unedited SD v1.4 model with the same prompts and seeds (the orig/ folder the script writes alongside the erased outputs), not against a real-image dataset. Regarding naming and implementation details, where variable names differ between the paper and the repository, please just follow the code. And yes, generation uses the diffusers==0.34.0 defaults for StableDiffusionPipeline with CompVis/stable-diffusion-v1-4 (PNDM scheduler, 50 steps, guidance 7.5, 512×512), fp16, safety checker disabled.
